### PR TITLE
Adding graph ML as a task on the hub

### PR DESCRIPTION
**Pull request by clefourrier**

### CODE REVIEW

**Review by osanseviero — APPROVED** [review]
Please make sure the indents in `const.ts` and `tasksData.ts` are consistent :) 
cc @gary149 and @mishig25 for the icon

**Comment by julien-c** [comment]
do you have a screenshot with the icon? :)

**Comment by clefourrier** [comment]
It was the [carbon:connect](https://icones.js.org/collection/carbon?s=connect) ([link to svg](https://api.iconify.design/carbon:connect.svg)) and I tried using the svelte download link.

**Comment by clefourrier** [comment]
Since the Graphormer model got merged into transformers, I'm planning to publish a blog post on graph classification with the hub on Monday 30, and it would be amazing if we could have graphs in the hub before: what can I do to make it happen?

**Comment by julien-c** [comment]
@gary149 do you want to create a quick icon for this new task? (or check the one from @clefourrier )

**Comment by gary149** [comment]
Updated icon to: 

hand drawn icon

**Comment by gary149** [comment]
What do you think of moving it to Multimodal category?

**Comment by julien-c** [comment]
yep i would put it in Multimodal (so we don't have any Other)

File: js/src/lib/interfaces/Types.ts [hunk 1]
```diff
@@ -602,6 +602,11 @@ export const PIPELINE_DATA = ensureRecordOfPipelines({
 		modality: "cv",
 		color:    "yellow",
 	},
+	"graph-ml": {
+		name:     "Graph Machine Learning",
+		modality: "other",
```
Comment (osanseviero) [inline js/src/lib/interfaces/Types.ts:607]:
```suggestion
		modality: "multimodal",
```

**Review by gary149 — APPROVED** [review]

**Comment by clefourrier** [comment]
How long after merging is the change integrated? Or should I add something else somewhere?

**Pull request merged**